### ISSUE TITLE
[Bug]: Impossible to interract with Three dots menu when u opened the menu through tabbing

### ISSUE THREAD

**Issue by efb4f5ff-1298-471a-8973-3d47447115dc** [FreeTubeApp/FreeTube#4022]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. Tab to the three dots menu
2. Open menu when in focus
3. Try to navigate the menu using arrow keys or tab
4. See that its not possible to navigate the menu

https://github.com/FreeTubeApp/FreeTube/assets/73130443/19018c00-3fad-4e54-a642-b0da74105037

### Expected Behavior

Users should be able to navigate the menu through the arrow keys or tabbing

### Issue Labels

accessibility issue, inconsistent behavior, usability issue

### FreeTube Version

v0.19.0-nightly-3413 Beta

### Operating System Version

Win10 22H2

### Installation Method

.exe

### Primary API used

Local API

### Nightly Build

- [X] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by MaxMaxMaxCh** [comment]
Anyone working on this? Willing to fix!

**Comment by jasonhenriquez** [comment]
No, not to my knowledge. Please go ahead, and ping us here if you need any support!
